src/STEMMUS_SCOPE_exe.m

```diff
@@ -25,8 +25,15 @@ function STEMMUS_SCOPE_exe(config_file, runMode)
                        'SiteProperties', ... % Site properties (e.g. lat, lon)
                        'fluxes', ... % Atmospheric fluxes
                        'TT', ... % Soil temperature over depth
-                       'SoilVariables', ... % Structure that includes different variables of soil moisture, added by Mostafa
-                       'GroundwaterSettings' ... % added by Mostafa
+                       'SoilVariables', ... % Structure that includes different variables of soil moisture
+                       'GroundwaterSettings' ... % groundwater settings including input data from MODFLOW
+                       'gwfluxes' ... % structure that includes groundwater recharge and its individual components

```

Comment:
We can add `gwfluxes = struct();` [here](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/7614318f77e251c414768b0fe4078c24875121b1/src/STEMMUS_SCOPE.m#L65), underthe section `% Get initial values`. I couldn't add this as a suggestion because this part of the code has not been changed yet. Can you please add this there?

Reply:
> We can add `gwfluxes = struct();` [here](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/7614318f77e251c414768b0fe4078c24875121b1/src/STEMMUS_SCOPE.m#L65), under the section `% Get initial values`. I couldn't add this as a suggestion because this part of the code has not been changed yet. Can you please add this there?
>

I added it now, thanks for the suggestion